source/reference/minio-server/settings/console.rst

```diff
@@ -0,0 +1,119 @@
+.. _minio-server-envvar-console:
+
+====================================================
+Environment Variables to Configure the MinIO Console
```

Comment:
IMO just say "MinIO Console Settings"

Since this is all nested under "Settings"